public/scripts/renderMapToDrawPolygon.js

```diff
@@ -16,6 +16,8 @@ const map = new ol.Map({
     center: ol.proj.fromLonLat([3.436, 55.3781]),
     zoom: 5,
   }),
+  ...(typeof isDetails !== 'undefined' &&
+    isDetails && { interactions: [], controls: [] }),
```

Comment:
May i know purpose of setting empty array based on condition?

Reply:
I assume this would be like a static map, where the user should have no ability to interact with it, such as drawing or zooming in or out. Based on the condition, I have passed an empty array to allow the map to initialize without any features. The media portal also behaves similarly.